packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -94,6 +94,7 @@ export default class ScriptTransformer {
             rootDir: this._config.rootDir,
           }),
         )
+        .update(filename)
```

Comment:
This isn't necessary because `getCacheKey` includes the `filename` already.

Reply:
Yes I thought so as well, my comment was only to seek confirmation ^^